hw/ip/otbn/util/otbn-ld

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+'''A wrapper around riscv32-unknown-elf-ld for OTBN
+
+This just adds the OTBN linker script and calls the underlying
+linker.'''
+
+import os
+import subprocess
+import sys
+
+
+def main() -> int:
+    ld_script = os.path.normpath(os.path.join(os.path.dirname(__file__),
+                                              '..', 'data', 'otbn.ld'))
+    if not os.path.exists(ld_script):
+        sys.stderr.write('Default linker script ({!r}) does not exist.'
+                         .format(ld_script))
+
+    try:
+        ld_name = 'riscv32-unknown-elf-ld'
```

Comment:
We'll probably need more logic to find the ld binary in the medium term; I just looked around, and we unfortunately don't have a standard environment variable to pick up the toolchain install path, or its name :-(

So let's leave it as-is for now, and hopefully fix that when we integrate this tooling into the build system at one point.

Reply:
Yeah, I did something similar and decided to punt on it for now :-)